conduct/COC_POLICY.md

```diff
@@ -59,7 +59,7 @@ A record of all communications related to an incident report, escalation request
 
 The identity of the harmed individual(s) and the and reporter(s) of an incident report must be kept confidential.
 
-The identity of the accused individual should be kept confidential. However, incidents requiring immediate action (e.g. to prevent further harm) and implementing the decision resulting from the incident management process might require sharing the identity of the accused individual beyond the people involved in the incident management process itself. In such cases, care should be taken to not make that sharing retaliatory.
+The identity of the accused individual should be kept confidential. However, incidents requiring immediate action (e.g. to prevent further harm) and implementing the decision resulting from the incident management process might require sharing the identity of the accused individual beyond the people involved in the incident management process itself. In such cases, care should be taken to not make that sharing retaliatory. The obligation of confidentiality towards the accused individul ends if the accused individul chooses to disclose the information publicly.
```

Comment:
hmm, this feels a bit "react patents clause" punitive - it's more that, any confidential information publicly disclosed stops requiring confidentiality, but any non-disclosed information continues to require it.

Reply:
I think that the argument here is should there be partial or incorrect information made public that needs to be addressed or corrected. This might require providing more context.

Reply:
(I hear your point, though.)

Reply:
The case where what’s made public is inaccurate/misleading is a tricky one, indeed, but in particular i think it’s important that the individual be able to reveal some parts of the confidential info while keeping other parts private.

Reply:
@ljharb fyi: I've reframed this a bit to try and better address your concern.

Reply:
awesome, i think this is good enough to go forward :-) it'll likely need refining in the future but the intention seems captured to me.